### ISSUE TITLE
Add Rule programmatically

### ISSUE THREAD

**Issue by gavelez** [ragunathjawahar/android-saripaar#127]
Hello, i'm writing a dynamic form generator for Android and i want to know how can i add a rule programmatically. I've searched on Google and StackOverflow and have not found something similar.

Thank you in advance

**Comment by ragunathjawahar** [comment]
You can use quick rules to programmatically add rules during runtime to the `Validator`.

**Comment by lepipele** [comment]
@ragunathjawahar I was trying to do this and I couldn't find an easy way to do it. Especially when I want to reuse existing rules you have in library rather than re-code them. This question on Stackoverflow essentially nails the essence of my question:

http://stackoverflow.com/questions/35737186/how-to-add-saripaar-validator-in-code-and-not-as-attribute

Let me know your stance on this - if needed I can do a pull request and add support for this in library; just would prefer to get clearance from you on if this is something you would want in library before I start coding.

**Comment by ragunathjawahar** [comment]
Right, we need to do something like [this](https://github.com/ragunathjawahar/android-saripaar/blob/v1/src/com/mobsandgeeks/saripaar/Rules.java) in v2. Feel free to send a PR. Follow the AOSP style guide for code style and look into the existing rules to see how existing rules are implemented.

**Comment by lepipele** [comment]
OK, let me see what I can do... I'll try to send first PR in as something small so that you can sign off on coding style and direction - the last thing I want is to spent lots of time on this and then you just reject it ;).

**Comment by ragunathjawahar** [comment]
That's the way to go :+1:

**Comment by gmann1** [comment]
Hey Ragunath,

I'm liking this library, but are there any updates on this issue?
